packages/bitcore-wallet-service/src/lib/chain/eth/index.ts

```diff
@@ -286,16 +286,14 @@ export class EthChain implements IChain {
 
   validateAddress(wallet, inaddr, opts) {
     const chain = 'ETH';
-    try {
-      Validation.validateAddress(
-        chain,
-        wallet.network,  // not really used for ETH. wallet.network is 'livenet/testnet/regtest' in wallet.
-        inaddr,
-      );
-    } catch (ex) {
+    const isValidAddress = Validation.validateAddress(
```

Comment:
is this a bug? (ie validateAddress does not throw?). Can you split this on a different PR, and add a test. Thanks.

Reply:
Reverted to master and will make a separate PR